crates/aptos-protos/proto/timestamp.proto

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package aptos.util.timestamp;
+
+message Timestamp {
+  // Represents seconds of UTC time since Unix epoch
+  // 1970-01-01T00:00:00Z. Must be from 0001-01-01T00:00:00Z to
+  // 9999-12-31T23:59:59Z inclusive.
```

Comment:
What does this mean
`Must be from 0001-01-01T00:00:00Z to 9999-12-31T23:59:59Z inclusive.`

Reply:
this is the standard of values that are allowed (this is stolen from original google definition); in a full u64, you can express post-heat-death-of-universe times, which break a LOT of downstream consumers which don't; year 9999 is a great end cap on that limit